manifest.json

```diff
@@ -15,5 +15,13 @@
     }
   ],
 
-  "permissions": ["*://www.quora.com/*"]
+  "permissions": ["*://www.quora.com/*"],
+
+  "background": {
+      "scripts": ["js/background.js"],
+      "persistent": false
+    },
```

Comment:
Per https://developer.chrome.com/extensions/event_pages#transition (see # 2 in that list), we shouldn't make this an event page until we switch to the alarms API.  I'd suggest removing from this PR if it's not critical to fixing #2 and then we can always fix in a future PR.